frontend/src/lib/components/modals/ErrorDetails.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { getTranslate } from '@tolgee/svelte';
+
+  import T from '$lib/components/T.svelte';
+  import { GenerateDebugInfo } from '$wailsjs/go/app/app';
+  import { BrowserOpenURL } from '$wailsjs/runtime/runtime';
+
+  export let parent: { onClose: () => void };
```

Comment:
Could take just onClose instead of the skeleton modal type, and also could be turned nullable to not have to pass it from the invalid installs error